test/phoenix/live_dashboard/live/metrics_page_test.exs

```diff
@@ -34,16 +34,6 @@ defmodule Phoenix.LiveDashboard.MetricsPageTest do
       live(build_conn(), "/dashboard/nonode@nohost/metrics?group=unknown")
   end
 
-  test "redirects to new node" do
-    {:ok, live, _} = live(build_conn(), "/dashboard/nonode@nohost/metrics?group=ecto")
-    send(live.pid, {:node_redirect, "foo@bar"})
-    assert_redirect(live, "/dashboard/foo%40bar/metrics?group=ecto")
-
-    {:ok, live, _} = live(build_conn(), "/dashboard/nonode@nohost/metrics?group=phx")
-    send(live.pid, {:node_redirect, "foo@bar"})
-    assert_redirect(live, "/dashboard/foo%40bar/metrics?group=phx")
-  end
-
```

Comment:
Now `PageLive` detects if a node is invalid, so this test doesn't work anymore. I feel like we need test involving real nodes, especially with the future custom pages, but I have 0 experience in this.